### PR TITLE
Fixes #26297 - graphql: add Host queries

### PR DESCRIPTION
**Pull request by kamils-iRonin**
This patch adds Host queries to the GraphQL api.

### CODE REVIEW

File: app/graphql/types/host.rb [hunk 1]
```diff
@@ -4,9 +4,19 @@ class Host < BaseObject
 
     global_id_field :id
     timestamps
-    field :name, String, null: false
+    field :name, String, null: true
+    field :build, Boolean, null: true
+    field :ip, String, null: true
+    field :ip6, String, null: true
+    field :path, resolver: Resolvers::Host::Path
```
Comment (timogoebel) [inline app/graphql/types/host.rb:11]: Can we also add `mac`, `last_report`, `domain_name`, `pxe_loader`, `enabled`, `uuid` here? They are low hanging fruit.

Comment (kamils-iRonin) [reply app/graphql/types/host.rb:11]: added

**Review by timogoebel — DISMISSED** [review]

**Comment by timogoebel** [comment]
We should add `environment` (#6594, ✅), `puppetclass` (#6607, ✅) and `compute_resource` (#6606, ✅) before merging this.

**Comment by timogoebel** [comment]
@kamils-iRonin: Please rebase this PR. All dependent PRs have been merged.

**Comment by kamils-iRonin** [comment]
[test foreman]

**Comment by timogoebel** [comment]
@kamils-iRonin: Test failures look related. I guess we need to sort the expected host IDs.

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks, @kamils-iRonin.

**Pull request merged**